src/System.Net.Http/src/System/Net/Http/Unix/CurlHandler.cs

```diff
@@ -512,39 +509,51 @@ private static bool AreEqualNetworkCredentials(NetworkCredential credential1, Ne
                 string.Equals(credential1.Password, credential2.Password, StringComparison.Ordinal);
         }
 
-        [Conditional(VerboseDebuggingConditional)]
-        private static void VerboseTrace(string text = null, [CallerMemberName] string memberName = null, EasyRequest easy = null, MultiAgent agent = null)
+        private static bool EventSourceTracingEnabled { get { return HttpEventSource.Log.IsEnabled(); } }
```

Comment:
Should the result of "IsEnabled()" be cached? We check this property often.

It just feels weird that we are wrapping a method call in a property. If "IsEnabled()" is a fast call, why isn't it a property?